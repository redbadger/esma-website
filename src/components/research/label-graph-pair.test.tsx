import React from "react";
import renderer from "react-test-renderer";
import LabelGraphPair from "./label-graph-pair";
import EarlyDevIcon from "../../svg/blocks.svg";

describe("LabelGraphPair", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<LabelGraphPair label="Test" graph={EarlyDevIcon} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
