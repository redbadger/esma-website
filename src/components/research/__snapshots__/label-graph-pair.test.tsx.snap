// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LabelGraphPair renders correctly 1`] = `
<div
  className="css-7hxiok-containerStyles"
>
  <div
    className="label-wrapper"
  >
    <span>
      Test
    </span>
  </div>
  <test-file-stub />
</div>
`;
